core.coffee

```diff
@@ -407,6 +407,19 @@
 			delete: (params) ->
 				return @request(params, method: 'DELETE')
 
+			batch: (data) ->
+				params =
+					method: 'POST'
+					url: '$batch'
+				overrides =
```

Comment:
Ok, in that case they need to be made to be expected and such that every http backend should be able to work with them, right now I'm guessing it'll only work with the `request` backend and not any in-browser one